resources-manager/include/ISystemResourceEntity.hpp

```diff
@@ -0,0 +1,29 @@
+// Concord
+//
+// Copyright (c) 2018-2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+namespace concord::performance {
+class ISystemResourceEntity {
+ public:
+  virtual ~ISystemResourceEntity() = default;
+  // Implementation returns how much more pruned blocks can this entity support.
```

Comment:
Is that comment relevant for the interface? Does this interface only relate to pruning?

Reply:
Will correct this to uint64.

Reply:
Done